io.github.Tahoma2D.yaml

```diff
@@ -0,0 +1,180 @@
+    cleanup:
+      - /etc
+      - /libexec
+      - /sbin
+    sources:
+    - type: archive
+      url: http://kernel.org/pub/linux/utils/kernel/hotplug/udev-175.tar.bz2
+      sha256: 4c7937fe5a1521316ea571188745b9a00a9fdf314228cffc53a7ba9e5968b7ab
+    - type: script
+      dest-filename: autogen.sh
+      commands:
+        - autoreconf -vfi
+
+  - name: libusb
+    config-opts:
+      - --disable-static
+    sources:
+      - type: archive
+        url: https://sourceforge.net/projects/libusb/files/libusb-1.0/libusb-1.0.21/libusb-1.0.21.tar.gz
+        sha256: 1a5b08c05bc5e38c81c2d59c29954d5916646f4ff46f51381b3f624384e4ac01
+
+  - name: freeglut
+    buildsystem: cmake-ninja
+    sources:
+      - type: archive
+        url: https://sourceforge.net/projects/freeglut/files/freeglut/3.0.0/freeglut-3.0.0.tar.gz
+        sha256: 2a43be8515b01ea82bcfa17d29ae0d40bd128342f0930cd1f375f1ff999f76a2
+
+  - name: superlu
+    buildsystem: cmake
+    config-opts:
+      - -DCMAKE_Fortran_COMPILER=/usr/lib/sdk/gfortran-62/bin/gfortran
+      - -DCMAKE_POSITION_INDEPENDENT_CODE=True
+    sources:
+      - type: archive
+        url: https://github.com/xiaoyeli/superlu/archive/v5.2.1.tar.gz
+        sha256: 77582501dedef295eb74e4dc9433e2816d2d8be211eae307379c13d93c65bc71
+
+  - name: openblas
+    no-autogen: true
+    make-args:
+      - DYNAMIC_ARCH=1
+      - FC=gfortran
+      - NO_LAPACKE=1
+    make-install-args:
+      - PREFIX=/app
+    sources:
+      - type: archive
+        url: https://github.com/xianyi/OpenBLAS/archive/v0.3.0.tar.gz
+        sha256: cf51543709abe364d8ecfb5c09a2b533d2b725ea1a66f203509b21a8e9d8f1a1
+
+  - name: lz4
+    buildsystem: cmake-ninja
+    subdir: contrib/cmake_unofficial
+    cleanup:
+      - /bin
+    sources:
+      - type: archive
+        url: https://github.com/lz4/lz4/archive/v1.8.2.tar.gz
+        sha256: 0963fbe9ee90acd1d15e9f09e826eaaf8ea0312e854803caf2db0a6dd40f4464
+
+  - name: lzo
+    cleanup:
+      - /bin
+      - /share/doc
+    sources:
+      - type: archive
+        url: https://www.oberhumer.com/opensource/lzo/download/lzo-2.10.tar.gz
+        sha256: c0f892943208266f9b6543b3ae308fab6284c5c90e627931446fb49b4221a072
+
+  - name: libmypaint
+    config-opts:
+      - --disable-gegl
+      - --disable-introspection
+    sources:
+      - type: archive
+        url: https://github.com/mypaint/libmypaint/releases/download/v1.3.0/libmypaint-1.3.0.tar.xz
+        sha256: 6a07d9d57fea60f68d218a953ce91b168975a003db24de6ac01ad69dcc94a671
+
+  - name: boost
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/boost
+      - mv -t /app/boost *
+    cleanup:
+      - /boost
+    sources:
+      - type: archive
+        url: https://sourceforge.net/projects/boost/files/boost/1.61.0/boost_1_61_0.tar.bz2
+        sha256: a547bd06c2fd9a71ba1d169d9cf0339da7ebf4753849a8f7d6fdb8feee99b640
+
+  - name: opentoonz
+    buildsystem: simple
+    build-commands:
+      - cd thirdparty/tiff-4.0.3 &&
```

Comment:
libtiff is in the runtime, why do you need to build it?

Reply:
Because like OpenToonz, T2D uses a custom build of libtiff with a patch to it.